quickstarts/microsoft.logic/logic-app-create/metadata.json

```
@@ -1,8 +1,8 @@
 {
   "$schema": "https://aka.ms/azure-quickstart-templates-metadata-schema#",
   "icon": "logic",  
-  "itemDisplayName": "Create a Consumption logic app workflow with an ARM template",
-  "description": "This template creates an empty Consumption logic app for defining a workflow.",
+  "itemDisplayName": "Create a logic app by using a template",
+  "description": "This template creates an empty logic app that can be used to define workflows.",
```

A Consumption logic app lets you create only one workflow versus a Standard logic app allows multiple workflows.

```suggestion
  "description": "This template creates an empty Consumption logic app to define a workflow.",
```

---

Missed one here.